Common/src/CConfig.cpp

```diff
@@ -7734,7 +7737,25 @@ CConfig::~CConfig() {
   delete[] Marker_CfgFile_SobolevBC;
   delete[] Marker_All_SobolevBC;
 
-  delete[] Marker_All_SendRecv;
+  delete[] Config_Filenames;
+
+  delete[] SST_Options;
+
+  if (IntInfo_WallFunctions != nullptr) {
+    for (iMarker = 0; iMarker < nMarker_WallFunctions; ++iMarker) {
+      if (IntInfo_WallFunctions[iMarker] != nullptr)
+        delete[] IntInfo_WallFunctions[iMarker];
+    }
+    delete[] IntInfo_WallFunctions;
+  }
+
+  if (DoubleInfo_WallFunctions != nullptr) {
+    for (iMarker = 0; iMarker < nMarker_WallFunctions; ++iMarker) {
+      if (DoubleInfo_WallFunctions[iMarker] != nullptr)
+        delete[] DoubleInfo_WallFunctions[iMarker];
+    }
+    delete[] DoubleInfo_WallFunctions;
+  }
```

Comment:
Bad merge, also not need to delete list-type options anymore #1579 
```suggestion
  delete[] Marker_All_SendRecv;
```